frontend/src/components/v2/Button/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import { ButtonHTMLAttributes, forwardRef, ReactNode } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+type Props = {
+  children: ReactNode;
+  isDisabled?: boolean;
+  // loading state
+  isLoading?: boolean;
+  // various button sizes
+  size: 'sm' | 'md' | 'lg';
+};
+
+export type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & Props;
+
+export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
+  ({ children, isDisabled = false, className, ...props }, ref): JSX.Element => {
+    return (
+      <button
+        ref={ref}
+        aria-disabled={isDisabled}
+        type="button"
+        className={twMerge(
```

Comment:
Why don't we use classnames (we already use it for some other components) instead of tw-merge? It is much more popular package and I'd be more confident in adding it. What do you think?

Reply:
I think the answer to that would be this. Try to override and the classname priority would hit u @mv-turtle


https://github.com/dcastil/tailwind-merge/blob/v1.8.1/docs/what-is-it-for.md